### DISCUSSION TITLE
Merge multiple YAML files with yq strict confinement installation

### DISCUSSION THREAD

**Discussion by bezydelj** [mikefarah/yq#1198] [Q&A] [answered]
Can anyone suggest how to merge multiple files when yq has no access to them because of snap install strict confinement?

This doesn't work:
`yq ea '. as $$item ireduce ({}; . * $$item )' file1 file2`

**Comment by mikefarah** [comment, marked as answer]
Have you seen the snap notes here: https://github.com/mikefarah/yq#snap-notes

You could do something like

```
yq ea '. as $$item ireduce ({}; . * $$item )' <(sudo cat file1) <(sudo cat file2)
```
